include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -283,6 +287,79 @@ double CMAES<SelectionPolicyType>::Optimize(
   return overallObjective;
 }
 
+// Transforms the candidate into the given bounds.
+template<typename SelectionPolicyType>
+void CMAES<SelectionPolicyType>::BoundaryTransform(arma::mat& iterate)
```

Comment:
So, if I'm understanding that this function, given `iterate`, will transform each element of iterate such that it falls between `lowerBound` and `upperBound`, you could just use `arma::clamp()`, I think.  Correct me if I'm wrong and there's a bit that I missed.  (It looks like maybe there is some handling for when `upperBound` and `lowerBound` are too close together?  I have not figured out exactly what that does.)